code-challenges.test.js

```diff
@@ -13,47 +15,175 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
-// Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
-const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
-// Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
+    const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+    // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+    const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+    // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
+
+
+    describe("arrayCap", () =>{
+        it("Capitalizes the first letter in every string within the array", () => {
+            const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+            const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+            
+            expect(arrayCap(randomNouns1)).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+            expect(arrayCap(randomNouns2)).toEqual(["Temperature", "Database", "Chopsticks", "Mango"])
+        })
+    })
+
+    // FAIL  ./code-challenges.test.js
+    // mult3
+    //   ✓ Takes in an array of values and multiplies each of them by 3 (2 ms)
+    // divisibleThree
+    //   ✓ checks if the value in the object is divisible by 3 (1 ms)
+    // arrayCap
+    //   ✕ Capitalizes the first letter in every string within the array
+    // ● arrayCap › Capitalizes the first letter in every string within the array
+    //   ReferenceError: arrayCap is not defined
+
 
 // b) Create the function that makes the test pass.
 
+  
+  const arrayCap = (arr) => {
+    return arr.map((string) => {
+        return string.charAt(0).toUpperCase() + string.slice(1)
+    })
+  }
+
+    //   PASS  ./code-challenges.test.js
+    //   mult3
+    //     ✓ Takes in an array of values and multiplies each of them by 3 (5 ms)
+    //   divisibleByThree
+    //     ✓ checks if the value in the object is divisible by 3 (1 ms)
+    //   arrayCap
+    //     ✓ Capitalizes the first letter in every string within the array (1 ms)
+    //   Test Suites: 1 passed, 1 total
+    //   Tests:       3 passed, 3 total
+
+
+
 // --------------------3) Create a function that takes in a string and logs the index of the first vowel.
 
 // Pseudo code:
+    // Input: a single string
+    // Output: a single number - the index value of the first vowel
+    // Process: define the values of the vowels for the functin to match against, and then use a for loop to check if the letters in the given strings are strictly equal to the vowels I specify.
```

Comment:
Could be explained a little better here; you could say you will make an array holding your vowels, and then check if each character in the string is included in your vowel array using the for loop.